packages/dicomImageLoader/src/imageLoader/convertColorSpace.ts

```diff
@@ -23,6 +23,25 @@ function convertYBRFull(imageFrame, colorBuffer, useRGBA) {
   }
 }
 
+/**
+ * This function checks color space conversion data requirements before
+ * apply them
+ * @param imageFrame
+ * @param RGBA
+ * @returns
+ */
+export function isColorConversionRequirementsFulfilled(imageFrame, RGBA) {
```

Comment:
According to the DICOM standard:
If images with Photometric Interpretation (0028,0004) YBR_FULL_422 or YBR_PARTIAL_422, are encoded with JPEG coding Process 1 (non hierarchical with Huffman coding), identified by DICOM Transfer Syntax UID "1.2.840.10008.1.2.4.50" the minimum compressible unit is YYCBCR, where Y, CB, and CR are 8 by 8 blocks of pixel values. The data stream encodes two Y blocks followed by the corresponding CB and CR blocks.

Thus, for JPEG encoded values, the columns is a multiple of 16, while the height is a multiple of 8, with the row length being a multiple of 2, as specified.
It is permissible to encode this as LEI YBR_FULL_422, so this would be just a straight multiple of 2 for the number of columns.

For YBR_FULL_420, the equivalent is:
Y Y
Y Y CB CR
So, the number of columns and number of rows must be a multiple of 2, AND the number of elements a multiple of 6/4  (eg 4 Y element, 1 CB and 1 CR for every 4 pixel elements)